pages/dkp/kommander/2.2/release-notes/2.2.0/index.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: layout.pug
+navigationTitle: DKP 2.2 Release Notes
+title: DKP 2.2 Release Notes
+menuWeight: 10
+excerpt: View release-specific information for DKP 2.2
+enterprise: false
+beta: false
+---
+
+**D2iQ&reg; Kommander&reg; (DKP&reg;) version 2.2 was released on April 6, 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+
+[Download](../download/) and [install](../install/) the latest version to get started.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download Kommander. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install this product.</p>
+
+## Release summary
+
+Welcome to D2iQ Kubernetes Platform (DKP) 2.2! This release continues our tradition of customer-led development making DKP even easier to deploy, supporting more platforms and applications, and includes our AI/ML solution, Kaptain, added as a catalog application. In this release, we are beginning the process of combining our two flagship products, Konvoy and Kommander, into a single DKP product with two service level options: DKP Enterprise for multi-cluster environments, and DKP Essentials for single-cluster environments.
+
+For this release, we maintain the Konvoy and Kommander documentation sets, while publishing some combined DKP documentation for processes, such as Upgrading DKP version.
+
+This version supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster you want to attach using Kommander 2.2 must be running a Kubernetes version in this range.
+
+## New features and capabilities
+
+These features and capabilities are new for Version 2.2
+
+### Upgrade catalog application via CLI and UI
+
+You can use either the CLI or the UI to [upgrade your catalog applications](../projects/applications/catalog-applications/#upgrade-catalog-applications).
+
+<p class="message--note"><strong>NOTE:</strong> Catalog applications must be upgraded to the latest version BEFORE upgrading the Kubernetes version (or Konvoy version for managed Konvoy clusters) on attached clusters, due to the previous versions' incompatibility with Kubernetes 1.22.</p>
+
+### Better integration with VMware vSphere
+
+You can use CAPI vSphere Provider while provisioning a DKP cluster on vSphere, which allows you to manage bootstrapping of VMs on a DKP cluster. This gives you improved productivity and speed of deploying VMs on DKP in a VMWare environment, including FIPS builds and air-gapped deployments.
```

Comment:
```suggestion
You can use CAPI vSphere Provider while provisioning a [DKP cluster on vSphere](../../../choose-infrastructure/vsphere/index.md), which allows you to manage bootstrapping of VMs on a DKP cluster. This gives you improved productivity and speed of deploying VMs on DKP in a VMWare environment, including FIPS builds and air-gapped deployments.
```

Reply:
@Colin1841 We should make it:
which allows you to manage bootstrapping of VMs for/in a DKP cluster.
DKP cluster is on a VM, not the other way round.

Reply:
Wouldn't the link for Kommander need to cross over to the Konvoy docs?

Reply:
Doh, yes - need to link to Kon docs. Do we need an actual link to that, vs relative link?